build-core-host-generic.sh

```diff
@@ -19,3 +19,9 @@ make -C "phoenix-rtos-filesystems" all
 
 b_log "Building phoenix-rtos-devices"
 make -C "phoenix-rtos-devices" all
+
+b_log "Building hostutils"
+make -C "phoenix-rtos-hostutils" -f Makefile.old "$CLEAN" all
```

Comment:
You shouldn't have to use Makefile.old 

That .old Makefile is workaround for building host utils when you pass TARGET other that host-generic-pc.

Reply:
On the second thought: default build with normal Makefile will enable sanitizers which can affect performance of tools. It may not be desirable

Reply:
I haven't considered anything you mentioned tbh - this is just copied from another target. But I think I shouldn't have used .old makefile in this case